backend/python/app/__init__.py

```diff
@@ -66,6 +66,8 @@ def create_app(config_name):
     )
 
     app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+    app.config["SQLALCHEMY_POOL_RECYCLE"] = 499
+    app.config["SQLALCHEMY_POOL_TIMEOUT"] = 20
```

Comment:
out of curiosity, did you find that you needed both the select 1 solution and these ping configs?